### ISSUE TITLE
Flexbox

### ISSUE THREAD

**Issue by addyosmani** [geppy/polymer-ie9-demo#1]
It's really great to see you exploring this work. I've been curious how well Polymer would work in IE9.I work with the Polymer team and am just wondering if you've run into any issues due to the lack of the latest flexbox being available.

Have you considered polyfilling it at all? I know that this is particularly tricky given that there isn't a fully prime-time polyfill for anything but the old spec available (Flexie/Re-flexie) but was curious if you had looked into this at all.

**Comment by geppy** [comment]
I hadn't looked into this at all, but I will now.  Thanks!

**Comment by geppy** [comment]
Created a branch “[flexbox](https://github.com/BrianGeppert/polymer-ie9-demo/tree/flexbox)” that uses [Reflexie](https://github.com/doctyper/reflexie) for CSS flexbox support.  I'm not seeing any improvement in this particular demo app.  I don't have any time to contribute to Reflexie at the moment, unfortunately, so I suppose I'll just try to check out the latest version every once in a while and see when the demo's layout starts working.

**Comment by geppy** [comment]
It looks like Reflexie isn't under active development.

**Comment by addyosmani** [comment]
Indeed. It looks like work on it has stalled. I'll see if it's possible to find another polyfill that is a little further along than Reflexie is. Would be nice to get the layout system working too. It may otherwise require quite a few layers for cross-browser styling.
